### PR TITLE
Stem/lancaster stemmer

### PR DESCRIPTION
**Pull request by LBenzahia**
I'll inspire the implementation from [nltk Lancaster stemmer](https://github.com/nltk/nltk/blob/develop/nltk/stem/lancaster.py).
#34

### CODE REVIEW

**Comment by kariminf** [comment]
Sorry, I didn't notice you have a problem when adding a new function. 

Apparently, the instructions outside describe are executed before the describes (the asynchronous mambo-jambo). Anyway, you can add an initialization function inside describe, like this:
```
//morpho.setCurrentStemmer("lancasterStemmer");
describe("English Morphology Lancaster Stemmer", function(){
    before(function(){
    morpho.setCurrentStemmer("lancasterStemmer");
  });
...
```
same thing for Porter:
```
describe("English Morphology porter stemmer ", function(){
    before(function(){
    morpho.setCurrentStemmer("porterStemmer");
  });
```

**Comment by LBenzahia** [comment]
Salam @kariminf,  ready to review

File: test/unit/eng.morpho_t.js [hunk 1]
```diff
@@ -115,7 +115,32 @@ describe("English Morphology porter stemmer ", function(){
...
+          expect(morpho.stem("multiply")).to.eql("multiply"); // No action taken if word ends with "-ply" 'multiply'
+          expect(morpho.stem("provision")).to.eql("provid"); // Replace "-sion" with "-j" to trigger "j" set of rules 'provid'
+          expect(morpho.stem("owed")).to.eql("ow"); // Word starting with vowel must contain at least 2 letters 'ow'
+          expect(morpho.stem("ear")).to.eql("ear"); //  ditto 'ear'
+          expect(morpho.stem("saying")).to.eql("say"); // Words starting with consonant must contain at least 3'say
+          expect(morpho.stem("crying")).to.eql("cry"); // letters and one of those letters must be a vowel 'cry'
+          expect(morpho.stem("string")).to.eql("string"); // ditto 'string'
+          expect(morpho.stem("meant")).to.eql("meant"); // ditto 'meant'
+          expect(morpho.stem("cement")).to.eql("cem"); // ditto 'cem'
+          //expect(morpho.stem("ness")).to.eql("nest"); // Change s to t 'nest' TODO: Make it change s to t 'nest'
```
Comment (LBenzahia) [inline test/unit/eng.morpho_t.js]: I've made that as a todo in #12

File: test/unit/eng.morpho_t.js [hunk 2]
```diff
@@ -115,7 +115,32 @@ describe("English Morphology porter stemmer ", function(){
...
+        });
+
+        /*it('Strip Prefixes', function(){ TODO: make it strip Prefixes
```
Comment (LBenzahia) [inline test/unit/eng.morpho_t.js]: I've made that as a todo in #12

**Comment by kariminf** [comment]
Salam @LBenzahia 
Thank you for the new function :)

I merged it if that's OK

**Pull request merged**